### DISCUSSION TITLE
How to proceed with "Correspondences" in Zootaxa

### DISCUSSION THREAD

**Discussion by Carol-Sokolowicz** [plazi/community#76] [Q&A]
Hi Donat,
Zootaxa has a lot of Correspondance issues that have no headings, although they are about taxonomic treatments, this is an exemple:

![image](https://user-images.githubusercontent.com/57491712/79605764-e14cad00-80c6-11ea-8a91-2309f6a7a4eb.png)
https://drive.google.com/open?id=1gFv33U8KYvNOQx364kbfgsjA3gZhlAvq

Should I make part of the title which has the taxon as a treatment heading so the rest of the text can be extracted as a treatment or not?

I have similar doubt about the erratum on a previous issue https://github.com/plazi/community/issues/17

I'm not processing both correspondence an erratum yet because of this doubt,

Cheers

**Comment by mguidoti** [comment]
@myrmoteras we need your input here

**Comment by myrmoteras** [comment]
When I processed early Zootaxa I did what you mentioned making either part of the title part of the treatment, or if possible the first mentioning of the name as the nomenclature relevant taxonomic name.

This is similar Checklist. 

Regarding erratum, that is a bit more complicated. I did this case by case, but often it is complicated. 
May be we can those which are not clear put aside, discuss at some point and then process them all together? What do you think @mguidoti @flsimoes ?

**Reply by mguidoti** [reply]
I honestly think we should have a way to connect the erratum with the original publication in GGI. Something that could be displayed on TB, Zenodo, future BLR website as well (like a direct link).

We should be able to indicate in GGI the precedent publication.

**Reply by flsimoes** [reply]
Not sure if it would be possible, but we could think of using the titles to signal their linking. Usually errata start with "ERRATUM: <Title of the original paper>".

**Reply by mguidoti** [reply]
That doesn't necessarily apply to all of the journals, but they do usually have a standard way (even if it's unique for one journal) to present errata. I guess what I'm saying is that we should have something on GGI to connect the precedent article, allowing for manually editing this link or individually processing errata, and something on the template creator that would teach GGI to identify errata and then search in TB for the precedent paper based on the available metadata.

**Reply by flsimoes** [reply]
Guess that would be very straightforward. Just need to have a field in Metadata for "connected UUID" or something like that. Could easily be shown on TB in the paper's page
